opea-comps/mega-service/example/orchestrator.py

```
@@ -0,0 +1,386 @@
+                    except Exception as e:
+                        raise e
+
+    @opea_telemetry
+    async def execute(
+        self,
+        session: aiohttp.client.ClientSession,
+        req_start: float,
+        cur_node: str,
+        inputs: Dict,
+        runtime_graph: DAG,
+        llm_parameters: LLMParams = LLMParams(),
```

Ruff (0.8.2)

Do not perform function call `LLMParams` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)